CHANGELOG.md

```diff
@@ -11,7 +11,7 @@
 ### Breaking Changes
 
 - Removed `az_storage_blobs.h`, including APIs related to storage service such as `az_storage_blobs_blob_client_init()` and `az_storage_blobs_blob_upload()`, and types such as `az_storage_blobs_blob_client` and `az_storage_blobs_blob_client_options`.
-  - These will ship in the upcoming 1.1.0 release and will continue to be available as preview from the following branch: https://github.com/Azure/azure-sdk-for-c/tree/feature/StorageBlobs
+  - These will ship in the upcoming 1.1.0 release and will continue to be available as beta from the following branch: https://github.com/Azure/azure-sdk-for-c/tree/feature/StorageBlobs
```

Comment:
Please don't change the changelog for releases before 1.0.0 (as that already shipped), they were using preview anyway.